apps/dashboard/app/helpers/application_helper.rb

```diff
@@ -64,9 +68,20 @@ def app_icon_tag(app)
   def icon_tag(icon_uri)
     if %w(fa fas far fab fal).include?(icon_uri.scheme)
       fa_icon(icon_uri.host, fa_style: icon_uri.scheme)
-    else
+    elsif !icon_uri.to_s.include? "svg"
       image_tag icon_uri.to_s, class: "app-icon", title: icon_uri.to_s, "aria-hidden": true
+    else
+      svg_tag(icon_uri.to_s, class: 'app-icon')
```

Comment:
svgs need titles and aria-hidden too.  Also you probably want `if icon_uri.to_s.end_with("svg")` instead of !include.